banners/mobile/components/BannerVar.vue

```diff
@@ -3,13 +3,13 @@
 		<MiniBanner
 			@close="onCloseMiniBanner"
 			@show-full-page-banner="onshowFullPageBanner"
-			@show-full-page-banner-preselected="onshowFullPageBannerPreselected"
+			@submit10Euro="submit10Euro"
```

Comment:
The [Vue style guide](https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing) recommends kebab-cased property names in the markup
```suggestion
			@submit-10-euro="submit10Euro"
```